enterprise/server/remote_execution/commandutil/commandutil_test.go

```diff
@@ -253,7 +253,7 @@ func TestRun_EnableStats_RecordsCPUStats(t *testing.T) {
 	require.Equal(t, "", string(res.Stderr))
 	require.Equal(t, 0, res.ExitCode)
 	require.GreaterOrEqual(
-		t, res.UsageStats.GetCpuNanos(), int64(2e9),
+		t, res.UsageStats.GetCpuNanos(), int64(1500e6),
```

Comment:
If you wanna go even lower like 1000e6 or something, I think that'd be fine too - would rather have looser confidence in this code than have it be flaky